src/images/schedule/grid.svg

```diff
@@ -0,0 +1,15 @@
+<svg preserveAspectRatio="none" viewBox="1170 -350 2640 1790" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Consider using `preserveAspectRatio="xMinYMax slice"`. I had the issue where the grid would stretch on mobile in my component as well.